src/Uno.UWP/Generated/3.0.0.0/Windows.Media.Capture/CameraCaptureUI.cs

```diff
@@ -2,47 +2,7 @@
 #pragma warning disable 114 // new keyword hiding
 namespace Windows.Media.Capture
 {
-	#if __ANDROID__ || __IOS__ || NET46 || __WASM__
```

Comment:
By removing those, they will come back on next generation, causing _noise_ commit.  Maybe you should just change them as the generator would do it?

Reply:
That's expected, and those will not come back, the generator will handle it properly.

Reply:
Ok.  Good to know.